app/src/main/res/layout/home_fragment.xml

```diff
@@ -20,24 +21,24 @@
             android:layout_marginTop="30dp"
             android:background="@drawable/frame"
             android:backgroundTint="#C6B2EA"
+            android:onClick="@{() -> viewModel.onD20Clicked()}"
             android:src="@drawable/d20_svg"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            android:onClick="@{() -> viewModel.onD20Clicked()}"/>
+            app:layout_constraintTop_toTopOf="parent" />
 
         <TextView
             android:id="@+id/textView"
```

Comment:
Clear names. This way you'll end up with 50 different textviews called 'textView' and mess up which one to use
Think about what the context is for this view

Whatever the context is, you can also make sure its unique by prefixing per example home_fragment_....